docs/nodes/db-neo4j-usage.md

```
@@ -0,0 +1,83 @@
+  "config": {
+    "uri_env": "NEO4J_URI",
+    "user_env": "NEO4J_USER",
+    "password_env": "NEO4J_PASSWORD",
+    "query": "MATCH (c:Company {name: $name}) RETURN c.industry, c.overall_severity"
+  }
+}
+```
+
+Environment variables:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)